agent_build_refactored/tools/run_in_ec2/boto3_tools.py

```diff
@@ -129,46 +129,51 @@ def create_and_deploy_ec2_instance(
         root_volume_size=root_volume_size
     )
 
-    # Try to establish SSH connection.
```

Comment:
Do we have any kind of period clean up process (like we had with libcloud) to make sure we periodically clean up any stale / stray instances?

Reply:
Thanks for clarification. I assume we also run / could run this job on a cron trigger so we don't have stray instances running a run in case previous build was interrupted and in case no new build is started for a while which would clean up old / stray instances.